# Network status: wifi signallevel and ethernet connection status

float32     wifi_signallevel
bool        ethernet_connected
